src/pkg/backup/details/details.go

```diff
@@ -503,14 +503,14 @@ func (ents entrySet) PrintEntries(ctx context.Context) {
 
 // MaybePrintEntries is same as PrintEntries, but only prints if we
 // have less than 15 items or is not json output.
-func (ents entrySet) MaybePrintEntries(ctx context.Context) {
-	if len(ents) > maxPrintLimit &&
-		!print.DisplayJSONFormat() &&
-		!print.DisplayVerbose() {
-		// TODO: Should we detect if the user is piping the output and
-		// print if that is the case?
-		print.Outf(ctx, "Restored %d items.", len(ents))
-	} else {
+func (ents entrySet) MaybePrintEntries(ctx context.Context, isRestore bool) {
+	if isRestore {
+		print.Outf(ctx, "Restored %d items", len(ents))
+	}
```

Comment:
I don't think we should introduce a bool. This print should be just a part of the restore code before calling the function.

Reply:
fair, moved it to the calling func.